packages/engine/src/datastore/arrow/element_conversion.rs

```diff
@@ -1,382 +0,0 @@
-#![allow(
-    clippy::too_many_lines,
-    clippy::cast_possible_wrap,
-    clippy::cast_sign_loss
-)]
-
-use arrow::{
-    array::{self, Array, ArrayRef},
-    datatypes::{self, ArrowNumericType, ArrowPrimitiveType, DataType, JsonSerializable},
-};
-use serde_json::value::Value;
-
-use super::{batch_conversion::col_to_json_vals, prelude::*};
-use crate::datastore::prelude::*;
-
-// TODO: UNUSED: Needs triage
-fn numeric_element_to_json_val<T: ArrowPrimitiveType + ArrowNumericType>(
```

Comment:
I think we should probably try and use [these methods](https://docs.rs/arrow/latest/arrow/json/index.html) if possible so I'd kinda prefer if we removed them and just added a comment linking back to how we did it before. 

I think if we leave them in, it'll push development back towards the old way of doing things rather than discovering the new conventions

Reply:
If you do a string search (not a Find Usages) for some of the methods you'll find them in a large portion of commented out code (not the best strategy but we chose to do it out of pragmatism) in the Rust runner. We should put a TODO there so we encounter it when we uncomment the code and see the errors